app/uk/gov/hmrc/bindingtariffrulingfrontend/views/search.scala.html

```diff
@@ -39,6 +39,7 @@
   @errorSummary(form)
 
   @helper.form(action = routes.SearchController.get()) {
+@helper.CSRF.formField
```

Comment:
This should work great, but I think we could also consider injecting [formWithCSRF](https://github.com/hmrc/play-frontend-govuk/blob/f7b1aef9bdbf451e9674611290b4d625b498b910/src/main/twirl/uk/gov/hmrc/govukfrontend/views/helpers/formWithCSRF.scala.html) from the play-frontend-govuk library and using that